src/cleo/_utils.py

```diff
@@ -70,9 +69,7 @@ def find_similar_names(name: str, names: list[str]) -> list[str]:
 
     # Only keep results with a distance below the threshold
     distance_by_name = {
-        key: value
-        for key, value in distance_by_name.items()
-        if value[0] < 2 * threshold
+        key: value for key, value in distance_by_name.items() if value[0] > threshold
```

Comment:
Maybe let's not do that filtering here, but in the `for` loop above, when inserting new items to `distance_by_name`?